Data_Engine/Create/LogicalAndRequest.cs

```diff
@@ -36,12 +36,22 @@ public static partial class Create
         [Description("Creates an IRequest that combines a couple of requests into a logical AND statement.")]
         [Input("request1", "First request in logical statement")]
         [Input("request2", "Second request in logical statement")]
-        [Output("LogicalAndRequest")]
+        [Output("request")]
         public static LogicalAndRequest LogicalAndRequest(IRequest request1, IRequest request2)
         {
             return new LogicalAndRequest { Requests = new List<IRequest> { request1, request2 } };
         }
 
         /***************************************************/
+
+        [Description("Creates an IRequest that combines a group of requests into a logical AND statement.")]
+        [InputFromProperty("requests")]
+        [Output("request")]
```

Comment:
Single string in the output attribute is the description, not the name.
![image](https://user-images.githubusercontent.com/53530468/84885142-887b9f80-b08a-11ea-9ade-eb21963ba8b4.png)